docs/vault.md

```diff
@@ -98,4 +109,13 @@ The key rotation steps are the following:
 
 1. Rotate the master key for the secondary nodes one by one.
 2. Step down the primary and wait for another primary to be elected.
-3. Rotate the master key for the previous primary node.
\ No newline at end of file
+3. Rotate the master key for the previous primary node.
+
+### Prevent key loss during rotation
```

Comment:
The `during rotation` phrase seems to be somewhat misleading. In fact, the mechanism for key loss prevention works during each new key creation. That is not only during the rotation but also when the node initializes and creates the first master encryption key.

Reply:
reworded